Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10.11-slim-buster
+LABEL maintaner="phaishuk"
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN mkdir -p /vol/web/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    django-user
+
+RUN chown -R django-user:django-user /vol/
+RUN chmod -R 755 /vol/web
+USER django-user
```

Comment:
I won't reject because of this, but remember to always add empty lines to the files